fixtures/backup/model_dependencies/sorted.json

```diff
@@ -15,12 +15,11 @@
   "sentry.DeletedOrganization",
   "sentry.Email",
   "sentry.Organization",
-  "sentry.User",
```

Comment:
This change in this PR reverts the change introduced in https://github.com/getsentry/sentry/pull/55744.